core-rust/state-manager/src/protocol/protocol_updates/protocol_update_definition.rs

```diff
@@ -89,7 +89,7 @@ impl ProtocolStateComputerConfig {
         ProtocolStateComputerConfig {
             network,
             validation_config: ValidationConfig::default(network_id),
-            costing_parameters: CostingParameters::default(),
+            costing_parameters: CostingParameters::babylon_genesis(),
```

Comment:
For consistency, I suggest we call this method `babylon_genesis` :)

Reply:
N/A anymore
(the `costing_parameters` are gone from here, and the remaining impl is all about `default`s)